lib/util/src/logging_util.c

```diff
@@ -3,24 +3,23 @@
  * Copyright (C) 2015-2021 Micron Technology, Inc.  All rights reserved.
  */
 
-#define MTF_MOCK_IMPL_logging_util
-#include "logging_util.h"
 #include <stdio.h>
 #include <syslog.h>
 
 #include <hse_util/logging.h>
-#include <hse_ikvdb/hse_gparams.h>
+
+#include "logging_util.h"
 
 void
-backstop_log(const char *fmt)
+hse_log_backstop(const char *fmt, ...)
 {
-    if (hse_gparams.gp_logging.destination == LD_SYSLOG) {
-        syslog(3, "%s: %s", __func__, fmt);
+    va_list ap;
+
+    va_start(ap, fmt);
+    if (hse_log_file) {
+        vfprintf(hse_log_file, fmt, ap);
     } else {
-        fprintf(logging_file, "%s: %s", __func__, fmt);
+        vsyslog(LOG_ERR, fmt, ap);
     }
+    va_end(ap);
```

Comment:
I nuked the mtf stuff here because it wasn't being used and because it choked ok putting HSE_NONNULL() on the function...  Crap, I should also use PRINTF_LIKE, will get it another day...